airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/logging/sentry.py

```diff
@@ -0,0 +1,107 @@
+import os
+import sentry_sdk
+import functools
+
+from dagster import OpExecutionContext, get_dagster_logger
+
+
+def setup_dagster_sentry():
+    """
+    Setup the sentry SDK for Dagster if SENTRY_DSN is defined for the environment.
+
+    Additionally TRACES_SAMPLE_RATE can be set 0-1 otherwise will default to 0.
+
+    Manually sets up a bunch of the default integrations and disables logging of dagster
+    to quiet things down.
+    """
+    from sentry_sdk.integrations.argv import ArgvIntegration
+    from sentry_sdk.integrations.atexit import AtexitIntegration
+    from sentry_sdk.integrations.dedupe import DedupeIntegration
+    from sentry_sdk.integrations.logging import LoggingIntegration, ignore_logger
+    from sentry_sdk.integrations.modules import ModulesIntegration
+    from sentry_sdk.integrations.stdlib import StdlibIntegration
+
+    ignore_logger("dagster")
+
+    SENTRY_DSN = os.environ.get("SENTRY_DSN")
+    SENTRY_ENVIRONMENT = os.environ.get("SENTRY_ENVIRONMENT")
+    TRACES_SAMPLE_RATE = float(os.environ.get("SENTRY_TRACES_SAMPLE_RATE", 0))
```

Comment:
i'm actually not sure if this is helpful for these - I have it disabled for now on `airbyte-ci` pipelines but 🤷🏼 

Did you see a use for it?

Reply:
The use generally is that Ill leave it set to 1.0 in production to get performance data and error rates.

We can change this to a bool, just figured Id leave it a more direct number